xmovie/core.py

```diff
@@ -441,44 +465,47 @@ def save(
         ----------
         filename : str
             Pathname to final movie/animation. Output is dependent on filetype:
-            Creates movie for `*.mp4` and gif for `*.gif`
-        remove_frames : Bool
-            Optional removal of frame pictures (the default is True; False will
+            creates movie for ``*.mp4`` and gif for ``*.gif``.
+        remove_frames : bool
+            Optional removal of frame pictures (the default is ``True``; ``False`` will
             leave all picture files in folder).
-        remove_movie : Bool
+        remove_movie : bool
             As `remove_frames` but for movie file. Only applies when filename
-            is given as `.gif` (the default is True).
-        progress : Bool
+            is given as `.gif` (the default is ``True``).
+        progress : bool
             Experimental switch to show progress output. This will be refined
-            in future version and currently only works with `parallel=False`
-            (the default value is False).
-        verbose : Bool
+            in future version and currently only works with ``parallel=False``
+            (the default value is ``False``).
+        verbose : bool
             Experimental switch to show output of ffmpeg commands. Useful for
             debugging but can quickly flood your notebook
-            (the default is False).
-        overwrite_existing : Bool
+            (the default is ``False``).
+        overwrite_existing : bool
             Set to overwrite existing files with `filename`
-            (the default is False).
-        parallel : Bool
-            Whether or not to use dask to save the frames in parallel.
+            (the default is ``False``).
+        parallel : bool
+            Whether or not to use Dask to save the frames in parallel.
         parallel_compute_kwargs : dict
-            Keyword arguments to pass to dask's `compute()` function.
+            Keyword arguments to pass to Dask's :func:`~dask.compute`.
         framerate : int
-            Frames per second for the output movie file. Only relevant for '.mp4' files.
-            (the default is 15)
+            Frames per second for the output movie file. Only relevant for ``.mp4`` files.
+            (The default is 15).
         ffmpeg_options: str
             Encoding options to pass to ffmpeg call.
-            Defaults to : `"-c:v libx264 -preset veryslow -crf 10 -pix_fmt yuv420p"`
-        gif_palette : Bool
+            Defaults to: ``"-c:v libx264 -preset veryslow -crf 10 -pix_fmt yuv420p"``.
+        gif_palette : bool
             Use a gif colorpalette to improve quality. Can lead to artifacts
-            in very contrasty situations (the default is False).
+            in very contrasty situations (the default is ``False``).
```

Comment:
Wow this is a really thorough correction all over the docstrings. Thank you so much for all the efforts.

Reply:
Welcome!